### DISCUSSION TITLE
dotnet core does not work

### DISCUSSION THREAD

**Discussion by DencoL** [mfussenegger/nvim-dap#695] [Q&A]
Hi,
I'm trying to make debugging for C# (.NET Core) projects to work. I'm on Windows.

I setup the adapter as described here: https://github.com/mfussenegger/nvim-dap/wiki/Debug-Adapter-installation#Dotnet

I set breakpoint (indicated by the 'B'). When I try to run the debugger, I'm prompted for the DLL. I provide the DLL, the app runs, but it runs in Production environment, the breakpoint changes to 'R' and the app runs normally without ever hitting the breakpoint.

Is there anything else I'm missing?

**Comment by mfussenegger** [comment]
Check the logs and make sure you build the project with debug symbols
